modules/cassandra/src/main/java/org/apache/ignite/cache/store/cassandra/utils/session/CassandraSessionImpl.java

```diff
@@ -281,8 +280,7 @@ else if (CassandraHelper.isPreparedStatementClusterError(e))
                     if (!CassandraHelper.isTableAbsenceError(e) && !CassandraHelper.isHostsAvailabilityError(e))
                         throw new IgniteException(errorMsg, e);
 
-                    if (log != null)
-                        log.warning(errorMsg, e);
+                    log.warning(errorMsg, e);
```

Comment:
It will produce NullPointerException if Ignite was started with Basic configuration without Spring support added (see the same comment above)